tests/sentry/tasks/test_clear_expired_snoozes.py

```diff
@@ -27,9 +28,14 @@ def test_simple(self, send_robust):
 
         clear_expired_snoozes()
 
-        assert Group.objects.get(id=group1.id).status == GroupStatus.UNRESOLVED
+        group1.refresh_from_db()
```

Comment:
If we need to do this here in our tests, should we not do the same in some parts of our code?

In other words, why is using `.get()` not correct?